ncolony/process_events.py

```diff
@@ -26,8 +27,11 @@ class Receiver(object):
     :params monitor: a ProcessMonitor
     """
 
-    def __init__(self, monitor):
+    def __init__(self, monitor, environ=None):
```

Comment:
If you put `environ=os.environ`, you'll get 100% branch coverage.

Reply:
...But a lint error :)

Reply:
:fire: